docs/faq.rst

```diff
@@ -280,6 +280,40 @@ I'm scraping a XML document and my XPath selector doesn't return any items
 
 You may need to remove namespaces. See :ref:`removing-namespaces`.
 
+SSL Errors on OS X
+------------------
+
+At the time of this writing (April 2016), the version of OpenSSL that
+ships on OS X El Capitan (10.11.4) is quite old. The system OpenSSL is
+version ``0.9.8zg`` whereas the latest version is ``1.0.2g``.
+
+The current version of Scrapy (1.0.5) also has issues that will
+be addressed in version 1.1, so we'll need to install the release
+candidate.
```

Comment:
@kmike , not sure I'm following. This is about limitations because of OS X 's OpenSSL version, that are still there for Scrapy 1.1.
Am I missing something?